src/frontend/js/components/SearchSuggestField/SearchSuggestField.tsx

```diff
@@ -224,8 +224,17 @@ export const SearchSuggestFieldBase = ({
     filterName: modelName.CATEGORIES | modelName.ORGANIZATIONS,
     payload: string,
   ) => {
+    const metaCategorySubpath = payload.substr(2, 8);
+
+    const filter = Object.values(filters)
```

Comment:
This is a negative point of hooks IMO. You can't test this implementation. You can't test `addFilter` function